Cargo.toml

```diff
@@ -195,9 +192,11 @@ ipnet = "2.9.0"
 itertools = "0.11.0"
 js-sys = "0.3.67"
 keccak = "0.1.4"
+lazy_static = "1.4.0"
```

Comment:
afair once_cell has all features of lazy_static. No need to have a new dep

Reply:
Replaced in this side PR: https://github.com/aspectron/rusty-kaspa/pull/78

@biryukovmaxim  Please review.